dashboard/app/models/level_source_image.rb

```diff
@@ -25,6 +25,12 @@ def save_to_s3(image)
     return false if CDO.disable_s3_image_uploads
     return false if image.blank?
 
+    begin
+      image = ImageLib::to_png(image)
```

Comment:
Mostly because this is lower work :). The s3_image_url helper method just assumes that everything (ie. everything already uploaded) is a png, so we'd have to do something like convert the existing ones to jpg (which might take forever), or store some "this level_source.id is when we changed to jpg" (which would require some coordination and/or a few minutes of downtime) or store it for individual level_source_images (adding db writes).

Also it is apparently not mandatory for browsers to support image/jpeg:
* https://github.com/scottjehl/Device-Bugs/issues/33
* http://www.w3.org/TR/2011/WD-html5-20110405/the-canvas-element.html#dom-canvas-todataurl
````
User agents must convert the provided type to ASCII lowercase before establishing if they support that type and before creating the data: URL.

When trying to use types other than image/png, authors can check if the image was really returned in the requested format by checking to see if the returned string starts with one of the exact strings "data:image/png," or "data:image/png;". If it does, the image is PNG, and thus the requested type was not supported. (The one exception to this is if the canvas has either no height or no width, in which case the result might simply be "data:,".)
````
  
(imagemagick will magickally figure out the image format from the image data itself, the requested mime type doesn't need to be sent)